designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -318,18 +318,33 @@ internal fun ContentDrawScope.render(
     val shapePaths =
         shape.computePaths(style, density, frameSize, customArcAngle, vectorScaleX, vectorScaleY)
 
-    val fillBrush =
-        style.background.mapNotNull { background ->
+    val customFillBrushFunction = customizations.getBrushFunction(name)
+    val customFillBrush =
+        if (customFillBrushFunction != null) {
+            customFillBrushFunction()
+        } else {
+            customizations.getBrush(name)
+        }
+
+    val fillBrush: List<Paint> =
+        if (customFillBrush != null) {
             val p = Paint()
-            val b = background.asBrush(document, density)
-            if (b != null) {
-                val (brush, fillOpacity) = b
-                brush.applyTo(frameSize, p, fillOpacity)
-                p
-            } else {
-                null
+            customFillBrush.applyTo(frameSize, p, 1.0f)
+            listOf(p)
```

Comment:
That's pretty simple.

Reply:
Yeah, it's nice how they added `RuntimeShader` and `ShaderBrush` -- made this super simple coz we were already using `Brush` everywhere anyway.